webrtc.html

```diff
@@ -1395,7 +1391,8 @@
                 which point the <code><a>RTCPeerConnection</a></code> can fully
                 adopt the pending local description, or rollback to the current
                 description if the remote side rejected the change.</p>
-                <p>When the method is invoked, the user agent must <a>set the
+                <p>When the method is invoked, the User Agent MUST return the
+                result of <a href="#set-description">setting the
                 RTCSessionDescription</a> indicated by the method's first
```

Comment:
Hmm, I take that back. It seems to consistently refer to the input argument.

I guess I tripped over it expecting "set X" to mean: X is what's being modified, rather than X is the input argument. Shouldn't this link be calle something like "set the local description"? Are we avoiding this because of the "current" vs "pending" wording difficulty?